### DISCUSSION TITLE
certificates when running on kubernetes.

### DISCUSSION THREAD

**Discussion by geohardtke** [nextflow-io/nextflow#2785] [Q&A]
Hi @pditommaso and community,

I'm experimenting with Nextflow and Kubernetes (on HPC), but are having issues with certificates. 
When I run the example (https://www.nextflow.io/docs/latest/kubernetes.html#execution) I get:

```bash
PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

Any hint would be appreciated!

**Comment by pditommaso** [comment]
This usually is related to the use of custom TLS certificates. Looks for more details on StackOverflow https://stackoverflow.com/questions/18378869/pkix-path-building-failed-in-java-application
